core/src/epicli/data/common/defaults/configuration/shared-config.yml

```diff
@@ -5,6 +5,8 @@ specification:
   custom_repository_url: '' # leave it empty to use local repository or provide url to your repo
   custom_image_registry_address: '' # leave it empty to use local registry or provide address of your registry (hostname:port). This registry will be used to populate K8s control plane and should contain all required images.
   download_directory: /tmp # directory where files and images will be stored just before installing/loading
+  vault_location: '' # if empty "BUILD DIRECTORY/vault" will be used
+  vault_tmp_file_location: SET_BY_AUTOMATION
```

Comment:
It seems `vault_tmp_file_location` is set only when `shared_config.specification.vault_location == ''`. What if someone will use non-default value for `vault_location`?

Reply:
Correct it should be always set